package.json

```diff
@@ -34,5 +34,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
```

Comment:
- [ ] I can see that you settle your react app but when running the app with `npm start` is not working and it brings this :point_down:  kindly fix it since I'm not able to run your app.  Your react-scripts is not found
![react](https://user-images.githubusercontent.com/88894525/204497244-84ad6fe4-179c-4185-878e-6b89fbe1de36.png)

Reply:
That's weird because I can run it without any issues.

Reply:
Did you do npm install?